CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+# Contributing to TIDES 
+Thank you for contributing to the TIDES Project.  This document defines the roles and process for contributing to the project and documents the governance roles and approach for decision-making.
+
+## Contents
+* [Roles](#roles)
+* [How to Contribute](#howto)
+* [How to Create Good Issues or Pull Requests](#good-pulls)
+* [Contributor Agreement](#contrib-agreement)
+* [Licenses](#licenses)
+* [Project Governance](#governance)
+* [Review and Approval Process](#approvals)
+* [Code of Conduct](#code-of-conduct)
+
+## [Roles](roles)
+There are two types of contributors to TIDES: 
+ 
+* Registered Contributors can create and respond to issues and can generate and comment on pull requests, and
+* All other Stakeholders can create and respond to issues.
+ 
+All contributors and stakeholders are asked to adhere to the [Code of Conduct] (#code).
+
+To become a registered Contributor, fill out the registration form at [this link][contributor-registration].
+
+## [How to Contribute](howto)
+Contributions should be offered through GitHub issues and pull requests. 
+ 
+By making any contribution to the projects, contributors self-certify to the [Contributor Agreement](#contrib-agreement).
+ 
+## [Contributor Agreement](contrib-agreement)
+ 
+By making any contribution to the projects, contributors self-certify to the following Contributor Agreement:
+ 
+By making a contribution to this project, I certify that:
+>  
+> a. The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or
+>  
+> b. The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or
+>  
+> c. The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.
+>  
+> d. I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.
+>  
+Attribution: This Contributor Agreement is adapted from the node.js project available here: https://github.com/nodejs/node/blob/main/CONTRIBUTING.md 
+
+## [How to Create Good Issues or Pull Request](howto)
+ 
+Content to be added. 
```

Comment:
Let's remove this, but add,
```md
## [How to write good commit messages](git-commit)
1. Separate subject from body with a blank line.
2. Limit the subject line to around 50 characters.
3. Capitalize the subject line.
4. Do not end the subject line with a period.
5. Use the imperative mood in the subject line, e.g., "fix schema file names", "define timestamp format".
6. Wrap the body at 72 characters.
7. Use the body to explain what and why.
8. [Reference issues or pull-requests on GitHub](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
```

Reply:
Suggest just referencing: https://www.conventionalcommits.org/en/v1.0.0/#summary or similar

Reply:
It's a little heavy handed, e.g., MUST use a prefix

Reply:
I have moved this best practices into the How to Commit section and removed the How to Create Good Issues and Pull Requests section.  If you want to make further edits to this section, please provide specific recommended text.